docker/Dockerfile.test

```diff
@@ -12,13 +12,14 @@ ENV SHORT_SHA $SHORT_SHA
 
```

Comment:
This Dockerfile is used by both the cached state rebuild, and the cached state test:
https://github.com/ZcashFoundation/zebra/search?q=Dockerfile.test

Keeping them consistent is a good idea, because it helps prevent bugs. But I just wanted to mention it in my review, because it's not documented in the Dockerfile, or mentioned in the PR.